packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -210,48 +195,27 @@ export abstract class LoadBalancedServiceBase extends cdk.Construct {
     this.assignPublicIp = props.publicTasks !== undefined ? props.publicTasks : false;
     this.desiredCount = props.desiredCount || 1;
 
-    // Load balancer
-    this.loadBalancerType = props.loadBalancerType !== undefined ? props.loadBalancerType : LoadBalancerType.APPLICATION;
-
-    if (this.loadBalancerType !== LoadBalancerType.APPLICATION && this.loadBalancerType !== LoadBalancerType.NETWORK) {
-       throw new Error(`invalid loadBalancerType`);
-    }
-
     const internetFacing = props.publicLoadBalancer !== undefined ? props.publicLoadBalancer : true;
 
     const lbProps = {
       vpc: this.cluster.vpc,
       internetFacing
     };
 
-    if (this.loadBalancerType === LoadBalancerType.APPLICATION) {
-      this.loadBalancer = new ApplicationLoadBalancer(this, 'LB', lbProps);
-    } else {
-      this.loadBalancer = new NetworkLoadBalancer(this, 'LB', lbProps);
-    }
+    this.loadBalancer = new ApplicationLoadBalancer(this, 'LB', lbProps);
 
     const targetProps = {
       port: 80
```

Comment:
I realize this isn't strictly part of the change, but do we want to allow this to be configurable as per a previous discussion? (See: https://github.com/aws/aws-cdk/issues/3666)

Reply:
I think it goes back to how much do we want to make configurable.